CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
-## NEXT
+## 0.2.7
```

Comment:
We're removing public APIs.
```suggestion
## 0.3.0
```

Reply:
Thanks. I updated version and updated callbacks package